### ISSUE TITLE
`test_views.AjaxSearchBarTest.test_cantus_id_search` fails 1 time out of 10 - we should rewrite it to fail less often

### ISSUE THREAD

**Issue by jacobdgm** [DDMAL/CantusDB#1193]
I just updated the staging server. `test_models` ran fine, but I'm getting a failure in `test_views` for one of the tests I recently wrote for #1188:
```
======================================================================
FAIL: test_cantus_id_search (main_app.tests.test_views.AjaxSearchBarTest)
----------------------------------------------------------------------
Traceback (most recent call last):
  File "/code/django/cantusdb_project/main_app/tests/test_views.py", line 5566, in test_cantus_id_search
    self.assertEqual(len(non_matching_chants), 0)
AssertionError: 1 != 0
```

**Comment by jacobdgm** [comment]
running it again, it passes:
```
root@da42ee733a02:/code/django/cantusdb_project# python manage.py test main_app.tests.test_views.AjaxSearchBarTest
Found 3 test(s).
Creating test database for alias 'default'...
System check identified no issues (0 silenced).
...
----------------------------------------------------------------------
Ran 3 tests in 0.464s

OK
Destroying test database for alias 'default'...
```
Maybe a cosmic ray hit the server running our Staging VM?

**Comment by jacobdgm** [comment]
hm... but running the entirety of `test_views`, I'm getting a failure again. My guess is that at https://github.com/DDMAL/CantusDB/blob/e2a9e5cf7299f0c5b7895465145ec23e2bc6d922/django/cantusdb_project/main_app/tests/test_views.py#L5541, `make_chant`, the randomly generated chant is being assigned a Cantus ID that starts with `0`.

**Comment by jacobdgm** [comment]
...and, running the entirety of `test_views` again, everything passes. I think this test will just fail 10% of the time, since there are only 10 possible characters that can start a Cantus ID. We should rewrite this test to fail less frequently.

**Comment by lucasmarchd01** [comment]
Fixed with #1278
